include/cpuinfo_x86.h

```diff
@@ -93,6 +94,12 @@ typedef struct {
 // This function is guaranteed to be malloc, memset and memcpy free.
 X86Info GetX86Info(void);
 
+// Calls cpuid and returns an initialized CacheInfo for x86 architecture.
```

Comment:
This is basically stating the function's signature, how about something shorter and more human friendly:
`Returns cache hierarchy informations`